widgets/numeric-refinement-list/numeric-refinement-list.js

```diff
@@ -72,7 +72,7 @@ function numericRefinementList({
         return option;
       });
 
-      let hasNoResults = facetValues.length === 0;
+      let hasNoResults = results.nbHits === 0;
```

Comment:
No because the facetValues are manually set thus they are always here (you always have facetValues)